numpy/distutils/fcompiler/intel.py

```diff
@@ -27,6 +28,19 @@ def runtime_library_dir_option(self, dir):
         return '-Wl,-rpath=%s' % dir
 
 
+    def get_version(self):
+        if platform.system() == 'Windows':
+            version_cmd = ["icl", "-dummy"]
+            completed_process = subprocess.run(version_cmd, stdout=PIPE, stderr=STDOUT)
+            status, output = completed_process.returncode, completed_process.stdout
+            version = re.search(r'Version\s*([\d.]+)', output).group(1)
+        else:
+            version_cmd = ["icc", "-dumpversion"]
+            completed_process = subprocess.run(version_cmd, stdout=PIPE, stderr=STDOUT)
+            status, output = completed_process.returncode, completed_process.stdout
+        return version
```

Comment:
This gives `NameError` on linux

Reply:
Oops. Serves me well to push without testing.

I squashed and pushed the locally tested change.